CondFormats/HcalObjects/src/HcalCondObjectContainerBase.cc

```diff
@@ -13,9 +13,9 @@ HcalCondObjectContainerBase::HcalCondObjectContainerBase(const HcalTopology* top
 }
 
 void HcalCondObjectContainerBase::setTopo(const HcalTopology* topo) {
-  if (topo && !topo->denseIdConsistent(packedIndexVersion_)) {
+  if ((topo) && (packedIndexVersion_ != 0) &&

```

Comment:
@bsunanda which use case  `(packedIndexVersion_ != 0) ` is supposed to take care of?